composer/datasets/imagenet.py

```diff
@@ -26,7 +26,7 @@
 class ImagenetDatasetHparams(DatasetHparams, SyntheticHparamsMixin):
     """Defines an instance of the ImageNet dataset for image classification.
 
-    Parameters:
+    Args:
```

Comment:
References to `DataSpec` in the "Returns" section for the `initialize_object` method aren't resolving.